components/core/Server.py

```diff
@@ -27,8 +27,8 @@ def serve_ui1():
 
 
 # download endpoints
-server.add_url_rule(rule='/download/start', endpoint='start', view_func=Download.start)
-server.add_url_rule(rule='/download/kill', endpoint='kill', view_func=Download.kill)
+server.add_url_rule(rule='/api/download/start', endpoint='start', view_func=Download.start)
+server.add_url_rule(rule='/api/download/kill', endpoint='kill', view_func=Download.kill)
```

Comment:
When we use ingress to route to Bassa services, we need `/api` in our endpoints to differentiate between client and api